pkg/providers/aws/cluster_network_provider.go

```diff
@@ -101,6 +103,7 @@ type NetworkManager interface {
 	DeleteNetworkPeering(*NetworkPeering) error
 	IsEnabled(context.Context) (bool, error)
 	DeleteBundledCloudResources(context.Context) error
+	getNonOverlappingDefaultCIDR(context.Context) (*net.IPNet, error)
```

Comment:
We don't export this, so I think we can remove it from the interface.

Reply:
@laurafitzgerald I think this can be removed

Reply:
👍  done